### PR TITLE
Fix esrpcodesigning task

### PR DESCRIPTION
**Pull request by apoorvdeshmukh**
This PR is created to investigate and update the esrpcodesigning task

### CODE REVIEW

**Review by stuartpa — APPROVED** [review]
:shipit:

File: build/azure-pipelines/build-product.yml [hunk 1]
```diff
@@ -80,63 +80,46 @@ stages:
...
           Pattern: 'sqlcmd.exe'
           signConfigType: 'inlineSignParams'
-          SessionTimeout: '600'
-          MaxConcurrency: '5'
-          MaxRetryAttempts: '5'
           inlineOperation: |
             [
               {
-                "keyCode": "CP-230012",
+                "keyCode": "CP-230072",
```
Comment (shueybubbles) [inline build/azure-pipelines/build-product.yml:92]: this is a test cert code, not a production cert code

File: build/azure-pipelines/build-product.yml [hunk 2]
```diff
@@ -80,63 +80,47 @@ stages:
...
+                  }
                 ],
                 "toolName": "signtool.exe",
                 "toolVersion": "6.2.9304.0"
-              },
-              {
-                "keyCode": "CP-230012",
-                "operationSetCode": "SigntoolVerify",
-                "parameters": [
-                {
```
Comment (apoorvdeshmukh) [inline build/azure-pipelines/build-product.yml:131]: @shueybubbles :This piece of code was not auto generated. Should I retain this "SigntoolVerify" block as well?

File: build/azure-pipelines/build-product.yml (hunk 1 above)
Comment (apoorvdeshmukh) [reply build/azure-pipelines/build-product.yml]: Updated to use production keycode.

File: build/azure-pipelines/build-product.yml (hunk 2 above)
Comment (apoorvdeshmukh) [reply build/azure-pipelines/build-product.yml]: Retained this code block. Verified the run [here](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=186071&view=logs&j=2524592f-1200-5da2-277a-7cff66fe62fe&t=0771f747-4673-5e21-eb1d-b95b586710a2)

**Pull request merged**